content/blog/pulumiup-global-cloud-iac-platform-engineering-ai-conference/index.md

```diff
@@ -0,0 +1,145 @@
+- **"[Industrialize the Configuration of Your GitHub Repositories With IaC](https://conference.pulumi.com/talks/industrialize-the-configuration-of-your-github-repositories-with-iac/)"** by Alexandre Nédélec (Software Engineer, Avanade) will live code using the GitHub, Azure native, and Azure AD Pulumi providers to create a GitHub repository and configure its pipeline to deploy to Azure using OpenID Connect.
+- **[Panel Discussion: "Infrastructure as Code — Can We Do Better?"](https://conference.pulumi.com/talks/infrastructure-as-code-can-we-do-better/)** featuring Joe Duffy (Pulumi), Brian Grant (Stealth), Elad Ben-Israel (Winglang), Adam Jacob (System Initiative), and Luke Hoban (Pulumi) will discuss the past, the present, and the future of Infrastructure as Code and the overall infrastructure technologies.
+
+## Workshops and Hands-On Learning
+
+Want to get hands-on experience? Our live workshops are designed to deepen your understanding of trending topics, Pulumi, and its applications.
+
+- **"[Managing Team Secrets with 1Password & Pulumi ESC](https://www.pulumi.com/resources/managing-team-secrets-with-1password-pulumi-esc/)"** on September 25, led by Phil Johnston (1Password) and Diana Esteves (Pulumi).
+- **"[Securing IaC Pipelines in Regulated Industries](https://www.pulumi.com/resources/securing-iac-pipelines-in-regulated-industries/)"** on September 26, led by Josh Kodroff (Pulumi) and Marina Novikova (AWS).
+- **"[Observability as Code for AI Apps with New Relic and Pulumi](https://www.pulumi.com/resources/observability-as-code-for-ai-apps-new-relic/)"** on October 30, led by Diana Esteves (Pulumi) and Harry Kimpel (New Relic).
+
+## Amazing Partners and Sponsors
+
+Thank you to our amazing partners for supporting PulumiUP!
+
+- **[AWS](https://aws.amazon.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** yadda yadda?
+- **[Google Cloud](https://cloud.google.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** yadda yadda?
+- **[New Relic](https://developer.newrelic.com/pulumi/get-started-pulumi/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** yadda yadda?
+
+## Build Your Schedule Now
+
+With so many compelling topics across AI/ML, Platform Engineering, DevOps, and Cloud and IaC, now is the time to [build your PulumiUP schedule](https://conference.pulumi.com/schedule/?utm_source=PulumiUP&utm_medium=web&utm_campaign=FY2025Q1_Event_PulumiUP). Choose the sessions that align with your goals and interests to get the most out of this unique virtual event. Whether you're looking to sharpen your skills, discover the latest innovations, or connect with industry leaders, PulumiUP has something for everyone.
+
+Haven’t registered yet? It’s not too late! Join 5,000 engineers worldwide in this groundbreaking event and gain the insights you need to drive innovation at your company. [Register now->](https://www.pulumi.com/pulumi-up/)
```

Comment:
should we mention that if they miss it, their will be recordings sent out? We can ask them to subscribe to the newsletter to get links to the recordings, that way even if they don't attend, we are still getting them into the database

Reply:
Good point. We actually tell them to register to get access to the recordings - it is definitely not timezone-friendly for many.